tensorflow/core/kernels/sparse_tensor_dense_matmul_op_gpu.cu.cc

```diff
@@ -160,4 +239,4 @@ DEFINE_ALL_INDEX_TYPES(complex128);
 
 }  // end namespace tensorflow
 
-#endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+#endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
May as well leave the newline on the end of the file. The removal of it creates an unnecessary difference to pay attention to. Plus, we (I) prefer a newline at the end of a file.

Reply:
Note to self: addressed.